### ISSUE TITLE
AG WG Review: 3.3.8 Accessible Authentication (Minimum) WCAG2ICT guidance

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#368]
## Summary
Review of proposed WCAG2ICT guidance for the new WCAG 2.2 SC:  3.3.8 Accessible Authentication (Minimum), as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
    * All SCs can be found in the file [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md).
    * The definition of "cognitive function test" is found in the file [comments-on-definitions-in-wcag-2.2-glossary-in-appendix-a.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-definitions-in-wcag-2.2-glossary-in-appendix-a.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will replace the 2013 Note, and it covers all three WCAG versions (2.0, 2.1, or 2.2).  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying SC 3.3.8 Accessible Authentication (Minimum) to Non-Web Documents and Software](https://w3c.github.io/wcag2ict/#applying-sc-3-3-8-accessible-authentication-minimum-to-non-web-documents-and-software).
* Section [Applying "cognitive function test" to Non-Web Documents and Software](https://w3c.github.io/wcag2ict/#applying-cognitive-function-test-to-non-web-documents-and-software)

## Reference Material
In case you need them, links to various resources on 3.3.8:

* [3.3.8 Accessible Authentication (Minimum)](https://www.w3.org/TR/WCAG22/#accessible-authentication-minimum) SC text in WCAG 2.2
* [Intent section of Understanding 3.3.8 Accessible Authentication (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication#intent)
* [cognitive function test](https://www.w3.org/TR/2023/REC-WCAG22-20231005/#dfn-cognitive-function-test) glossary definition in WCAG 2.2

**Comment by nitedog** [comment]
Based on [Understanding SC 3.3.8](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html#auth-approaches), this SC is based on the assumption that there are password managers (e.g. built into web browsers or available on desktop). This does not always apply to software due to operating system limitations outside the control of the content author. An example is a device that requires authentication during first installation to customize the device to your own profile but does not yet have the operating system fully loaded to allow password managers to work. I suggest we add a note to that effect.

**Comment by maryjom** [comment]
Regarding the comment from @nitedog (so the AG TF can see what actions have been occurring on this topic raised:

The TF surveyed the proposed changes (see [survey results](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-public-comments-R4/results#xq7)) and discussed in last week's meeting ([see 16 May minutes](https://www.w3.org/2024/05/16-wcag2ict-minutes.html#t11)). A [resolution on 3.3.8](https://www.w3.org/2024/05/16-wcag2ict-minutes.html#r08) was reached to not include the note at that time. 

Per Shadi's request, we will revisit the proposal in the 22 May meeting to enable him to participate in the discussion since he was unable to make the 16 May call.

**Comment by maryjom** [comment]
The WCAG2ICT TF is surveying a new draft note for 3.3.8 Accessible Authentication to address the comment from @nitedog. The currently proposed note appears as Note 4 in the [Applying SC 3.3.8 Accessible Authentication (Minimum) to Non-web Documents and Software](https://deploy-preview-378--wcag2ict.netlify.app/#applying-sc-3-3-8-accessible-authentication-minimum-to-non-web-documents-and-software) which is a part of pull request 378 being surveyed by the WCAG2ICT TF this week.

**Comment by maryjom** [comment]
The TF made a [resolution to add a note to 3.3.8](https://www.w3.org/2024/05/30-wcag2ict-minutes#r08) in the 30 May meeting that addresses Shadi's concerns from his [comment posted on 17 May](https://github.com/w3c/wcag2ict/issues/368#issuecomment-2117705943) above. The text is as follows:

>NOTE 4: There are cases where non-web software has an authentication process and no alternative or assistance mechanism is feasible, for example when entering a password when starting, powering on / turning on an ICT (device or otherwise). In such situations, it may not be possible for the non-web software to meet this success criterion.

**Comment by maryjom** [comment]
The Note 4 change above has been incorporated into the draft. Closing this issue, since the AG WG review is done. WCAG2ICT document is now in CfC to publish that ends on 1 July.
